rust/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
   * usrsctp snapshot 4e06feb01cadcd127d119486b98a4bd3d64aa1e7.
   * wingetopt 1.00.
 * `WebRtcServer`: A new class that brings to `WebRtcTransports` the ability to listen on a single UDP/TCP port (PR #834, PR #845).
+* Minor API breaking changes
```

Comment:
Probably a missing dot.

Reply:
This changelog is inconsistent with dots already, probably not worth it :upside_down_face:, I wish there was a linter that could insert or remove them automatically in consistent way.

Reply:
I never found that rule 🙁